cranelift/codegen/src/isa/aarch64/abi.rs

```diff
@@ -295,6 +295,10 @@ fn load_stack_from_fp(fp_offset: i64, into_reg: Writable<Reg>, ty: Type) -> Inst
 }
 
 fn store_stack(mem: MemArg, from_reg: Reg, ty: Type) -> Inst {
```

Comment:
What is the difference between this and `store_stack_sp`?

Reply:
`store_stack_sp` may generate several instructions to lower the SP addition.